### DISCUSSION TITLE
starlette.Response compatibility with httpx.Response

### DISCUSSION THREAD

**Discussion by filiphanes** [encode/starlette#2039] [Ideas] [answered]
Hi,
since starlette and httpx are created by same authors, wouldn't it be great to have compatible Request, Response objects?
At least same attributes. I have migrated my code from aiohttp to httpx to have more consistent conventions and naming, but now I see its not so consistent.

For example:
starlette.Response uses .body and .media_type attributes, and httpx.Response uses .content and .content_type attributes.

From reading sources it looks like they may not have exactly same meaning in some cases, but it would be nice to have property with same meaning.

Thanks

**Comment by Kludex** [comment, marked as answer]
It's not possible.
